src/SourceBuild/content/build.sh

```diff
@@ -226,6 +226,9 @@ function Build {
       # kill off the MSBuild server so that on future invocations we pick up our custom SDK Resolver
       "$CLI_ROOT/dotnet" build-server shutdown
 
+      # Point MSBuild to the custom SDK resolvers folder, so it will pick up our custom SDK Resolver
+      export MSBUILDADDITIONALSDKRESOLVERSFOLDER="$scriptroot/artifacts/toolset/VSSdkResolvers/"
+
```

Comment:
Do we need this in build.cmd too?

Reply:
No, we don't care about prebuilts on Windows and we don't build the SourceBuild SDK Resolver before the outerbuild on Windows. We only need this on the Linux build based on the existing build scripts.